src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -458,6 +459,23 @@ class BuildArgument(cli.Argument):
             'required': False,
         },
     )
+    BETA_GROUP_NAMES_REQUIRED = cli.ArgumentProperties(
+        key='beta_group_names',
+        flags=('--beta-group',),
+        type=str,
+        description='Name of your Beta group',
+        argparse_kwargs={
+            'nargs': '+',
+            'metavar': 'beta-group',
```

Comment:
This needs `'required': True`. Otherwise it isn't required and will possibly break action invocation.